.psh.yaml.dist

```diff
@@ -48,6 +48,7 @@ const:
   DEVPORT: "8080"
   STOREFRONT_PROXY_PORT: 9998
   VERSION:
+  DOCKER_STOP_BEHAVIOUR: down
```

Comment:
please use the default stop, otherwise it would be breaking :wink:

Reply:
The current behaviour is `down`, so I defaulted it to `down`

Reply:
you are right :smile: was looking at the wrong diff, shown by github :see_no_evil: